locale/pt_pt/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,185 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Why `pt_pt` and not `pt`?

Reply:
No real reason, just that Wagtail itself uses `pt-pt` for Portuguese (Portugal): https://github.com/wagtail/wagtail/blob/7286c530e97e2eb64f3747c6904a7456789f82b7/wagtail/admin/localization.py#L34

and that we've already configured the settings to use `pt-pt`:

https://github.com/wagtail/guide/blob/8689a4ec7aacc8bef1034c6a91e3cad9a30ff24c/apps/guide/settings/base.py#L242

Django does use `pt` though:

https://github.com/django/django/blob/main/django/conf/locale/__init__.py#L455


P.s. `pt-pt` still works because `get_language_info()` falls back to `pt`

Reply:
I'd always go for single language codes as `pt-pt` doesn't provide a fallback for `pt-br` and `pt` does. Same for any other language/region combination where the language is the same as the region.

FYI: Wagtail had `nl_nl` and `nl` and it was very confusing with nasty side effects.

On top of that, shorter URLs are nicer. ;)

Reply:
Okay, I've renamed our config to use `pt` for Portuguese (Portugal).